data/saas/dataset/marigold_engage_dataset.yml

```diff
@@ -0,0 +1,48 @@
+dataset:
+  - fides_key: <instance_fides_key>
+    name: Marigold Engage Dataset
+    description: A sample dataset representing the Marigold Engage integration for Fides
+    collections:
+      - name: user
+        fields:
+          - name: activity
+            fidesops_meta:
+              data_type: object
+            fields:
+              - name: create_time
+                data_categories: [system.operations]
+                fidesops_meta:
+                  data_type: string
+          - name: engagement
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
+          - name: keys
+            fidesops_meta:
+              data_type: object
+            fields:
+              - name: sid
+                data_categories: [system.operations]
+                fidesops_meta:
+                  data_type: string
+              - name: cookie
+                data_categories: [user.device.cookie_id]
+                fidesops_meta:
+                  data_type: string
+              - name: email
+                data_categories: [user.contact.email]
+                fidesops_meta:
+                  primary_key: True
+                  data_type: string
+          - name: lists
+          - name: optout_email
+            data_categories: [system.operations]
+            fidesops_meta:
+              data_type: string
+          - name: smart_lists
+          # the vars here could be holding any kind of data, I'm not sure how best to cope with that
```

Comment:
Try not to leave any comments like these in the config, we can make a note in the spec doc